utils/alsoft-config/CMakeLists.txt

```diff
@@ -56,3 +56,4 @@ else()
         )
     endif()
 endif()
+target_include_directories(alsoft-config PRIVATE "${OpenAL_BINARY_DIR}")
```

Comment:
This seems to break the build if `alsoft-config` isn't getting built. It should probably be in a `if(TARGET alsoft-cobfig)` check, or put it in the two blocks that declare the target (one for Qt5 and another for Qt4).

Reply:
Thanks!
I think I will just duplicate the line in both cases, like it is done very everything else.
Ideally, it will have a patch to deduplicate everything here in the future.